File: b_js_solution/0404_js/basic.js
function solution(n) {
    if (n < 90) return 1;
    else if (n === 90) return 2;
    else if (90 < n && n < 180) return 3;
    else return 4;
}

const n = [70, 91, 180, 90];

// print
n.forEach(item => {
    console.log(solution(item));
})

File: c_py_solution/0404_py/basic.py
def solution(n):
    if n < 90:
        return 1
    elif n == 90:
        return 2
    elif 90 < n < 180:
        return 3
    else:
        return 4

n = [70, 91, 180, 90]
for i in n:
    print(solution(i))
